doc/nrf/libraries/emds/emds.rst

```diff
@@ -0,0 +1,52 @@
+.. _emds_readme:
+
+Emergency data storage
+######################
+
+.. contents::
+    :local:
+    :depth: 2
+
+The emergency data storage (EMDS) library provides functionality for storing of data with a guaranteed time, based on the amount of data that are being stored.
+It can be used in situations of critical power shortage to store data in memory before the battery that powers the device is about to fully discharge.
+
+.. note::
+    The Power-fail comparator for the nRF528xx can not be used with this feature, as it will prevents the NVMC from performing write operations to the flash.
+
+Overview
+********
+The :kconfig:`CONFIG_EMDS` enables the emergency data storage. The application must initialize by using :ref:`emds_init`, the pre-allocated flash area used to store data in.
+The :kconfig:`CONFIG_EMDS_SECTOR_COUNT` defines how many sectors that should be used to store data. The space set aside shall be large than all the data stored. The larger value will increase the flash spaced used, but will reduce the wear of the flash area.
+This has to be balanced between amount of data stored, flash available and how of it will call the :ref:`emds_store`. It should not be necessary to have a large value, as there should be a limited set of data stored this way, as time is of essence here.
```

Comment:
```suggestion
The :kconfig:`CONFIG_EMDS` enables the emergency data storage. The application must initialize the pre-allocated flash area by using :ref:`emds_init`.
The :kconfig:`CONFIG_EMDS_SECTOR_COUNT` defines how many sectors that should be used to store data. The allocated storage space must be larger than the combined data stored by the application. Choosing a larger flash area will demand more resources, while simutaniously increase the life expectancy of the flash. The choosen size shuold reflect the amount of data stored, the available flash resources, and how the application calls the :ref:`emds_store`. In general it should not be necessary to allocate a large flash area, since there should be a limited set of data stored to ensure that writing the flash on shutdown completes swiftly.
```